framework-docs/modules/ROOT/pages/attributes.adoc

```diff
@@ -1,20 +1,20 @@
 // Spring Portfolio
-:docs-site:                     https://docs.spring.io
-:docs-spring-boot:              {docs-site}/spring-boot/docs/current/reference
-:docs-spring-gemfire:           {docs-site}/spring-gemfire/docs/current/reference
-:docs-spring-security:          {docs-site}/spring-security/reference
+// :docs-site:                     https://docs.spring.io
```

Comment:
Do we still need this file?

In other words, can we move `chomp` and `fold` to `antora.yml` and remove this one?